lib/options.js

```diff
@@ -20,6 +19,16 @@ function override(options, overrides) {
   }
 }
 
+function getShareAbleConfigPath(options) {
+  if (!options || !options.extends) {
+    return null;
+  }
+
+  return path
+    .resolve(process.cwd(), 'node_modules',
+      'min-wd-config-' + options.extends);
```

Comment:
While I guess this will work most of the time, it would be nice to use node's require algorithm to resolve the path. I'd prefer using https://www.npmjs.com/package/resolve for this.

Reply:
Thanks for the tip, will fix.

Reply:
Fixed